<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>259</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/window/assets/emby.png</normaloff>:/window/assets/emby.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>410</width>
     <height>200</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>Emby Server</string>
    </attribute>
    <widget class="QCheckBox" name="chk_https">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Https verwenden</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_address">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_port">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_username">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Benutzer Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_username">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_password">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Passwort</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_password">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>Kollektionen</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="lbl_maxActors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. Anzahl Schauspieler</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_maxDirectors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. Anzahl Regisseure</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_maxStudios">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. Anzahl Studios</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_maxGenres">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. Anzahl Genres</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_maxActors">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_maxDirectors">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_maxStudios">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_maxGenres">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3">
    <attribute name="title">
     <string>Optionen</string>
    </attribute>
    <widget class="QLabel" name="lbl_fontname">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Schriftart</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fontsize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Schriftgr√∂sse</string>
     </property>
    </widget>
    <widget class="QFontComboBox" name="cbx_fontfamily">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spb_fontsize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>8</number>
     </property>
     <property name="maximum">
      <number>14</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
